k8s/aat/common/ccd/data-store-api.yaml

```diff
@@ -24,8 +24,8 @@ spec:
       environment:
         CCD_DM_DOMAIN: ^https?://(?:api-gateway\.preprod\.dm\.reform\.hmcts\.net|dm-store-aat\.service\.core-compute-aat\.internal(?::\d+)?)
         CCD_CONDITIONAL_APIS_CASE_ASSIGNED_USER_AND_ROLES_ENABLED: true
-        CCD_S2S_AUTHORISED_SERVICES_CASE_USER_ROLES: aac_manage_case_assignment,fpl_case_service,iac,finrem_case_orchestration,unspec_service
-        DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_ps,probate_backend,divorce_ccd_submission,sscs,sscs_bulkscan,cmc,cmc_claim_store,cmc_claim_external_api,jui_webapp,pui_webapp,bulk_scan_orchestrator,fpl_case_service,iac,finrem_ccd_data_migrator,finrem_case_orchestration,employment_tribunals,ethos_repl_service,ccpay_bubble,ctsc_work_allocation,em_ccd_orchestrator,xui_webapp,bulk_scan_payment_processor,pcq_consolidation_service,em_npa_app,ecm_consumer,aac_manage_case_assignment,unspec_service,am_role_assignment_service,wa_task_configuration_api,nfdiv_cms,nfdiv_cos
+        CCD_S2S_AUTHORISED_SERVICES_CASE_USER_ROLES: aac_manage_case_assignment,fpl_case_service,iac,finrem_case_orchestration,unspec_service,divorce_frontend
+        DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_ps,probate_backend,divorce_ccd_submission,sscs,sscs_bulkscan,cmc,cmc_claim_store,cmc_claim_external_api,jui_webapp,pui_webapp,bulk_scan_orchestrator,fpl_case_service,iac,finrem_ccd_data_migrator,finrem_case_orchestration,employment_tribunals,ethos_repl_service,ccpay_bubble,ctsc_work_allocation,em_ccd_orchestrator,xui_webapp,bulk_scan_payment_processor,pcq_consolidation_service,em_npa_app,ecm_consumer,aac_manage_case_assignment,unspec_service,am_role_assignment_service,wa_task_configuration_api,nfdiv_cms,nfdiv_cos,divorce_frontend
```

Comment:
What does this name link to? Why can't we use div_cos or divorce_orchestration?

Reply:
no, it's divorce_frontend. Yes, it's very weird.

https://github.com/hmcts/div-case-orchestration-service/blob/9a26aecfc31ed5cbe023bc2052d2e9188602e717/src/main/resources/application.yml#L459

I have also added this to debug:
```
String s2stoken = authTokenGenerator.generate();
log.info("s2s Token: {}", s2stoken);
```

outcome:
```
10:11:30.038 INFO  [http-nio-4012-exec-4] u.g.h.r.d.o.s.CcdDataStoreService - s2s Token: Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJkaXZvcmNlX2Zyb250ZW5kIiwiZXhwIjoxNjExNzUzMDU2fQ.l2LRgT9FIFQoaF7O5XRMCKTyMN8EudhMqUQn9KRG0hPY1YvriKDbcYnnQP_Gp_07kVYFy2s_h8cMB1SEfo0SUA
```

jwt.io:
```
{
  "sub": "divorce_frontend",
  "exp": 1611753056
}
```

also confirmed with @Dizrupto. He was adding div-cos to PRD whitelist. The same.

Reply:
Ok. I guess it's not this story's job to change this.

Reply:
Yes, I will leave it as it is. It's been like this for ages.